README_VERITABLE.md

```diff
@@ -43,21 +43,62 @@ The project aims to enable supply chains to share insights and data across multi
 
 The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI (http://localhost:3000/swagger) that is provided with the server. The OpenAPI spec can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
 
-Bellow you will find commands for starting up the containers in docker (see the [Using Docker](#using-docker-easiest) section) or via cli (see [Via Cli](#via-cli)).
+Bellow you will find commands for starting up the containers in docker (see the [Using Docker](#using-docker-easiest) section).
 
 > The OpenAPI spec is generated from the model classes used by Aries Framework JavaScript. Due to limitations in the inspection of these classes, the generated schema does not always exactly match the expected format. Keep this in mind when using this package. If you encounter any issues, feel free to open an issue.
 
 > In case `npm i` fails with the references to the `node-gyp`. Please install xcode utilities by ruuning -> `brew install xcode`. Before executing `npm i` delete your node_modules in case it has some old references `rm -rf /node_modules`
 
-### Rest Client Args
+### Rest Client Envs
 
-The CLI args are defined under `src > cli.ts ` They are used to start up a service.
+The Envs are defined under `src > env.ts ` They are used to start up a container. They mostly have defaults and if you wish to overwrite these, provide them under `environment` in docker compose. For any envs that are an array of strings please provide them coma-separated like so: `- ENDPOINT=http://charlie:5002,ws://charlie:5003`.
 | Argument added | Required | Default | Description |
 | :------------- | :------: | :------ | :------- |
-| ipfs-origin | Y | http://localhost:5001 | IPFS endpoint |
-|opa-origin |N |http://localhost:8181 |OPA endpoint |
-|persona-title | N |"Veritable Cloudagent"|Tab name which you can see in your browser |
-|persona-color |N |"white" |Defines the background colour of swagger documentation|
+| LABEL| Y|"AFJ Rest"|Label for the agent|
+|WALLET_ID| Y| "walletId"| An id of the Agent's wallet|
+|WALLET_KEY| Y|"walletKey"| A key for the Agent's wallet|
+|ENDPOINT|Y|['http://localhost:5002', 'ws://localhost:5003']| An array of endpoint for the agent app|
+|LOG_LEVEL|Y| info| Log level for the app |
+|USE_DID_SOV_PREFIX_WHERE_ALLOWED|N|false|Allows the usage of 'sov' prefix in DIDs where possible|
+|USE_DID_KEY_IN_PROTOCOLS|N|true| Allows the use of DID keys in protocols|
+|OUTBOUND_TRANSPORT|Y|['http', 'ws']|Specifies the type of outbound transport|
+|INBOUND_TRANSPORT|Y|"[{"transport": "http", "port": 5002}, {"transport": "ws", "port": 5003}]"|Specifies the inbound transport, needs to be provided as a JSON parseable string|
```

Comment:
I actually prefer this as json exactly like you've done. Can you check though with @dblane-digicatapult that this isn't too horrible from an infra PoV